src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -40,14 +40,14 @@ public function __construct(array $headers = array())
      */
     public function __toString()
     {
-        if (!$this->headers) {
+        if (!$headers = $this->all()) {
```

Comment:
the property is still used.

In the normal HeaderBag, ``->all()`` is a simple getter for it. But it is now overwritten in ResponseHeaderBag to add the set-cookie header